README.md

```diff
@@ -149,6 +149,7 @@ These resources haven't specified any formal terms of use or licenses.
 
 A collection of illustration resources which contain a mixture of historical archive, contemporary and public domain assets.
 
+* [Biodiversity Heritage Library](https://www.flickr.com/people/biodivlibrary/) - Nearly 150,000 exquisite, historical illustrations of life on Earth. A mix of Public Domain [:copyright:](https://creativecommons.org/publicdomain/mark/1.0/) and Attribution (CC BY 2.0) [:copyright:](https://creativecommons.org/licenses/by/2.0/) licenses.
```

Comment:
Please place the copyright before the description for consistency with the rest of the list.